rpcs3/Emu/Memory/vm.h

```diff
@@ -396,15 +403,14 @@ namespace vm
 	}
 
 	void close();
-
-	u32 stack_push(CPUThread& CPU, u32 size, u32 align, u32& old_pos);
```

Comment:
Why would you move them? It causes error on both clang and gcc since this functions are defined after their use in template now.